### PR TITLE
Tailored flows: refresh the logo cropping tool

### LINKED ISSUE #67848: Task: Freshen up the logo editor in Tailored Flows
### Details

We have a beautiful new design for the logo cropping UI!

https://www.figma.com/file/BcUdhLlhAp7UHtQHNTI4zA/v1.-Tailored-Onboarding-%26-Launchpad?node-id=6487%3A140523#266620410

If we can keep the changes all in CSS and only [here](https://github.com/Automattic/wp-calypso/tree/trunk/client/components/site-icon-with-picker), that would be terrific, if not, we'll need to make sure we don't change the editor in Calypso. To see it in Calypso, you can go to /me and try to change your avatar.

### Checklist

_No response_

### Related

_No response_

### PR DESCRIPTION
**Pull request by renancarvalho**
## Proposed Changes

(WIP) Add a new design for the cropping icon tool. 
@alshakero, I tried to implement most of the design changes without change much of the component behaviour, It is not 100% how the design is, but I am afraid of doing too many changes on the imageCropping component. Let me know what you think. This is still a Work in Progress.

## Testing Instructions
- Pull this branch and run yarn start
- Navigate to `/setup/intro?flow=link-in-bio or flow=newsletter`
- Go to setup step and check the image upload tool

Related to #67848
Fixes #67848

### CODE REVIEW

File: client/components/site-icon-with-picker/style.scss [hunk 1]
```diff
@@ -1,16 +1,121 @@
+@import "@wordpress/base-styles/breakpoints";
+@import "@wordpress/base-styles/mixins";
+
 $icon-size: 80px;
 $icon-border-radius: 50%; /* stylelint-disable-line declaration-property-unit-allowed-list */
 
-.site-icon-with-picker__image-editor {
+.site-icon-with-picker__background {
+	min-width: calc(100vw - 16px);
+	min-height: 85vh;
```
Comment (alshakero) [inline client/components/site-icon-with-picker/style.scss:9]: Let's add `svh` for iPhones.

**Review by alshakero — CHANGES_REQUESTED** [review]
Let's add an `svh` for every `vh` rule.

File: client/components/site-icon-with-picker/style.scss [hunk 2]
```diff
@@ -1,16 +1,125 @@
...
+	}
+
+	@media screen and (orientation: landscape) and ( max-height: 720px ) {
+		min-width: 80vw;
+		min-height: 80vh;
+	}
+
+	@include break-medium {
+		min-width: 50vw;
+		min-height: 60vh;
```
Comment (alshakero) [inline client/components/site-icon-with-picker/style.scss]: vh.

**Review by alshakero — CHANGES_REQUESTED** [review]
One more vh.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7621157

Hi @renancarvalho, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include this string: ` Apply `

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7621157).

**Pull request merged**